CodeConverter/Shared/SolutionConverter.cs

```diff
@@ -112,9 +120,9 @@ private ConversionResult ConvertSolutionFile()
             };
         }
 
-        private static (string Find, string Replace, bool FirstOnly) GetProjectGuidReplacement(string projFilename, string contents)
+        private static (string Find, string Replace, bool FirstOnly) GetProjectGuidReplacement(string projPath, string contents)
         {
-            var projGuidRegex = new Regex(projFilename + @""", ""({[0-9A-Fa-f\-]{32,36}})("")");
+            var projGuidRegex = new Regex(projPath + @""", ""({[0-9A-Fa-f\-]{32,36}})("")");
```

Comment:
Will this still work for projects directly in the solution directory? Perhaps we can extract this static method to somewhere like ProjectFileTextEditor, and run unit tests on it more like https://github.com/icsharpcode/CodeConverter/blob/master/Tests/LanguageAgnostic/ProjectFileTextEditorTests.cs

It might be that just prefixing the regex to look for any of `/ \ "` is sufficient for this example. Though I can certainly see that someone could name a project the same and put it in a different directory too, which the current solution handles better. Since there are a few possible cases I think unit tests will help a lot

Reply:
I think for projects inside the sln dir this will fail. I'll change it to use a relative path instead. 

Yes, the case you're describing is why I chose to include the dirname as well instead of just the dir separator.